setup.py

```diff
@@ -1,34 +1,70 @@
 # Copyright 2017-2020 Lawrence Livermore National Security, LLC and other
-# Hatchet Project Developers. See the top-level LICENSE file for details.
+# CallFlow Project Developers. See the top-level LICENSE file for details.
 #
 # SPDX-License-Identifier: MIT
 
+import os
 from setuptools import setup, find_packages
-from callflow import __version__
 
+# ------------------------------------------------------------------------------
+# get the version safely!
+from codecs import open
+
+version = {}
+vfile = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "callflow", "version.py"
+)
+with open(vfile) as fp:
+    exec(fp.read(), version)
+version = version["__version__"]
+
+# ------------------------------------------------------------------------------
+# gather the data to be copied
+def list_files(directory):
+    paths = []
+    for (path, directories, filenames) in os.walk(directory):
+        files = [os.path.join(path, f) for f in filenames]
```

Comment:
variable `files` is not being used by the function, can we remove it?

Reply:
I think we need to copy only the data, and not the `.callflow` files. I think we can add a statement to check the type of files that can be considered as well. Otherwise, it might copy a lot of files.

Reply:
yes, looks like a copy-paste issue. get rid of files.
Yes, to .callflow too. we should see if os.walk returns hidden files/directories. i didnt encounter this.